common/djangoapps/terrain/ui_helpers.py

```diff
@@ -43,8 +43,10 @@
 
     # Content - Outline
     # Note that calling your org, course number, or display name, 'course' will mess this up
+    # Waiting for the factories to load - they load everything else through dependencies.
+    # Specifying them individually as they are independent from each other.
     re.compile(r'^Course Outline \|'): [
-        "js/base", "js/models/course", "js/models/location", "js/models/section"],
+        "js/base", "factories/base.js", "factories/course.js", "factories/outline.js"],
```

Comment:
Nit: it looks like ``js/base`` is included in ``factories/base.js``. It might make this slightly clearer to drop that one.